### ISSUE TITLE
lablgtk (2&3) are not working on macOS 11 (BigSur)

### ISSUE THREAD

**Issue by correnson** [garrigue/lablgtk#120]
The application randomly crashes with Segmentation Fault, not always but very often, typically when:
 - the window loses focus
 - a menubar item is clicked
 - exiting the application with ^Q

Observed with : why3 ide (gtk2) , frama-c-gui (gtk2) and coqide (gtk3).

**Comment by garrigue** [comment]
Just to be sure: are you using the X11 based versions of gtk 2/3 or the native ui ones?

**Comment by correnson** [comment]
I'm using gtk ports from Homebrew. These are probably native ui ports, since XQuartz is not launched by the gui-s.

```
$ brew info gtk+
gtk+: stable 2.24.32 (bottled), HEAD
GUI toolkit
https://gtk.org/
/usr/local/Cellar/gtk+/2.24.32_3 (1,194 files, 61.2MB) *
  Poured from bottle on 2020-11-19 at 11:50:31
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/gtk+.rb
==> Dependencies
Build: gobject-introspection ✔, pkg-config ✔
Required: atk ✔, gdk-pixbuf ✔, hicolor-icon-theme ✔, pango ✔
==> Options
--HEAD
	Install HEAD version
==> Analytics
install: 7,266 (30 days), 22,099 (90 days), 79,501 (365 days)
install-on-request: 3,416 (30 days), 10,298 (90 days), 27,465 (365 days)
build-error: 0 (30 days)
```

**Comment by correnson** [comment]
I confirm : looking closer to the gtk+ brew formula, it seems to be linked against `gdk-quartz` which, as far as I understand, is a port of `gdk` on Quartz, the native ui framework of macOS.

**Comment by garrigue** [comment]
I have eventually installed Big Sur, and tried running CoqIDE-8.12 (downloaded from the Coq website), and see no such problem.
Are these problem persisting? Can you reproduce them with official builds?
I didn't get reports from the Coq people, which seems surprising if the problem is frequent.

**Comment by correnson** [comment]
At least, it is still an issue with Why-3 IDE and Coq IDE compiled from opam.

**Comment by correnson** [comment]
Hmmm **UPDATE** I just there are new homebrew packages available for big-sur.
Installing…

**Comment by correnson** [comment]
Seems to be good now...

**Comment by garrigue** [comment]
Thanks for the update. I am also using Big Sur, but with macports, so it's hard to follow homebrew status.
